src/containers/signup/Signup.tsx

```diff
@@ -1,30 +1,44 @@
 import React from 'react';
-import './signup.less';
-import { Button, Col, Form, Input, Row, Typography } from 'antd';
-import { ParagraphProps } from 'antd/lib/typography/Paragraph';
-import { Link } from 'react-router-dom';
+import { Col, Row, Typography } from 'antd';
 import { Helmet } from 'react-helmet';
 import GreetingContainer from '../../components/greeting-container/GreetingContainer';
 import { signup } from '../../auth/ducks/thunks';
 import { connect, useDispatch } from 'react-redux';
 import { UserAuthenticationReducerState } from '../../auth/ducks/types';
 import { C4CState } from '../../store';
-import { BLACK, TEXT_GREY } from '../../colors';
+import { WHITE, BLACK, LIGHT_GREY } from '../../colors';
 import styled from 'styled-components';
+import MobilePageHeader from '../../components/mobile-pageheader/MobilePageHeader';
+import SignupForm from '../../components/signup-form/SignupForm';
+import useWindowDimensions, {
+  WindowTypes,
+} from '../../components/window-dimensions';
 
-const { Paragraph, Title } = Typography;
+const { Title } = Typography;
 
-const Footer: typeof Paragraph = styled(Paragraph)<ParagraphProps>`
-  color: ${TEXT_GREY};
-  line-height: 1.5;
+const SignupContainer = styled.div`
```

Comment:
`SignupContainer`, and Login, are a bit misleading to read. We're defining the SignupContainer in this file as a React component, so there is a duplicated meaning.

Reply:
Changed to `SignupPageContainer`, `MobileSignupPageContainer`, `LoginPageContainer`, etc.